src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -474,6 +484,16 @@ private function applyShipmentTransitionOnOrder(OrderInterface $order, $transiti
      * @param OrderInterface $order
      * @param string $transition
      */
+    private function applyPaymentTransitionOnOrder(OrderInterface $order, $transition)
+    {
+        $payment = $order->getLastPayment();
+        $this->stateMachineFactory->get($payment, PaymentTransitions::GRAPH)->apply($transition);
```

Comment:
Actually we should iterate through all payments of an order and apply transition on all of them.